### ISSUE TITLE
Duplicate comment in AST when comment is inside JSXElement

### ISSUE THREAD

**Issue by ArnaudBarre** [oxc-project/oxc#2674]
```tsx
const foo = <div
    // comment
/>
```

[playground](https://oxc-project.github.io/oxc/playground/?code=3YCAAICjgICAgICAgICxG0qZRraXI4KRyqLYDI4hhbsuvaes8MOF7P6cvEe6by7f5OoRf39hroCA)

**Comment by overlookmotel** [comment]
My guess is this might be related to lookahead. If the comment token gets re-lexed, it could end up inserted twice. I actually thought I saw Boshen push a commit to fix exactly that quite recently, but maybe it persists in some form.

**Comment by ArnaudBarre** [comment]
Yeah this was https://github.com/oxc-project/oxc/pull/2600 I think

**Comment by overlookmotel** [comment]
Thanks. #2600 looks like it should have solved that problem. So very possible my guess was incorrect. 🤷‍♂️
